src/components/WebexSettings/Tabs.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Tab from './Tab';
+
+/**
+ * Tabs component
+ *
+ * @param {object} props  Data passed to the component
+ * @param {string} props.tabs  The tabs to be rendered
+ * @param {string} props.selected  Specifies which tab is active
+ * @param {Function} props.onSelect  A callback to be executed on selecting active tab
+ *
+ * @returns {object} JSX of the component
+ */
+
+export default function Tabs({tabs, selected, onSelect}) {
+  const {content} = tabs.find((tab) => tab.key === selected);
+
+  return (
+    <div className="tabs">
+      <ul className="tabs-list">
+        {
+          tabs.map((tab) => (
+            <Tab
```

Comment:
is this component necessary? it looks simple and only used internally, so why not just put the tab heading here